infra/dev/laravel/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# use Debian Buster (10) with FPM
+FROM php:7.4-fpm
+
+# unlock php deb package
+RUN rm /etc/apt/preferences.d/no-debian-php
+
+ARG USER_ID
+ARG GROUP_ID
+
+RUN groupadd -f $GROUP_ID
```

Comment:
Is this to solve the writes problems we had previously?

Reply:
these line create a user with the same level of access as the local user on the host, we found that for solving the permission problem